openapi/2023-12.yaml

```diff
@@ -365,6 +428,35 @@ components:
                       type: string
                       example: ""
   schemas:
+    Flow:
+      type:
+        - object
+      required:
+        - type
+        - attributes
+      properties:
+        type:
+          type: string
+          example: "flows"
+        attributes:
+          type: object
+          required:
+            - id
+            - name
+            - tags
+          properties:
+            id:
+              type: string
+              description: ID of flow
+              example: "655637c97825e6ae655fe1b5"
+            name:
+              type: string
+              description: Name of flow
+              example: "Welcome flow"
+            tags:
+              type: array
+              description: Flow tags
+              example: ["automated", "welcome"]
```

Comment:
I'd add one more property to the list, whether the flow is active or not. That could be useful.

Other than that it looks good.

Reply:
It's a bit more verbose than that. I was thinking about it but there are different statuses for automated flows and campaigns. Are we ok with exposing our [statuses](https://github.com/ghostmonitor/sequence/blob/master/src/api/models/sequence.model.ts#L37)?

Reply:
Yes, that is what I thought about.